src/components/TaskListItem.js

```diff
@@ -0,0 +1,181 @@
+import DataStore from '../modules/DataStore.js';
+import Task from '../modules/Task.js';
+
+export const refreshTaskList = () => {
+  const tasks = document.querySelectorAll('.task');
+  tasks.forEach((prevTask) => {
+    prevTask.parentNode.removeChild(prevTask);
+  });
+
+  DataStore.tasks = DataStore.updateTaskIndex();
+  DataStore.tasks.forEach((task) => {
+    // eslint-disable-next-line
+    renderTaskItem(task);
+  });
+};
+
+export const isTaskCompleted = () => {
+  const completedTasks = document.querySelectorAll('input[name="isCompleted"]');
+  completedTasks.forEach((btnIsCompleted) => {
+    btnIsCompleted.addEventListener('click', () => {
+      if (btnIsCompleted.checked) {
+        btnIsCompleted.checked = true;
+        btnIsCompleted.parentNode.nextSibling.nextSibling.style.textDecoration = 'line-through';
+      } else {
+        btnIsCompleted.checked = false;
+        btnIsCompleted.parentNode.nextSibling.nextSibling.style.textDecoration = '';
```

Comment:
- [ ] still, the task renders false if the check-box is clicked, it doesn't return true

![Capture d’écran 2022-01-27 101146](https://user-images.githubusercontent.com/61562626/151327998-f64e8169-e77d-4906-8567-95d0052bd26f.png)